### DISCUSSION TITLE
Non-cascading deletion of Application from app of apps

### DISCUSSION THREAD

**Discussion by djaustin** [argoproj/argo-cd#10148] [Q&A]
I am hoping to remove an Application from an app of apps without the underlying resources being deleted (i.e. non cascading delete). I am unable to trigger this delete in the Argo UI as the app of apps keeps it in place. Deleting the Application from the git repo referenced in the app of apps causes the Application to be deleted **along with** all underlying resources. 

Is there a way to cause a non-cascading delete of the Application referenced within the App of Apps?  

Thanks!

**Comment by kostis-codefresh** [comment]
Removing the app from the GUI will never work, because as you already say ArgoCD will always take what is in Git and re-apply

Removing the app from Git should work. Can you check if you have finalizers on your manifests (both child app and parent app)?
